packages/ui-components/src/Input/TextInput.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { BaseInput, InputProps } from "./BaseInput";
+import { InputPrefix, InputWrapper } from "./helpers";
+
+export type TextInputProps = Omit<InputProps<string>, "type">;
+
+export const TextInput: React.FC<TextInputProps> = props => {
+  const { prefixIcon, disabled, invalid } = props;
+  return (
+    <InputWrapper disabled={disabled} invalid={invalid}>
+      <InputPrefix>{prefixIcon}</InputPrefix>
```

Comment:
I don't want to hold up this PR any longer, but wanted to leave a note to come back to. Should we conditionally render this prefixIcon only if an icon is provided? Will React be smart enough to not render empty DOM elements?